app/controllers/bright_events_controller.rb

```diff
@@ -0,0 +1,52 @@
+class BrightEventsController < ApplicationController
+  def index
+    @events = BrightEvent.all
+  end
+
+  def show
+    @event = BrightEvent.find(params[:id])
+  end
+
+  def new
+    @events = BrightEvent.new
```

Comment:
This is ok, but you are repeating this line many times, instead you can use the before_action filter.

See the documentation at:
https://guides.rubyonrails.org/action_controller_overview.html#filters